docs/manual/about_dds/qos_provider.rst

```diff
@@ -0,0 +1,413 @@
+
+QoS Policies
+------------
+
+The fields in a Qos policy are described in XML using a 1-to-1 mapping with the equivalent IDL representation in the DDS specification. For example, the Reliability Qos policy is represented with the following structures:
+
+.. code-block:: C
+
+	struct Duration_t {
+		long sec;
+		unsigned long nanosec;
+	};
+	struct ReliabilityQosPolicy {
+		ReliabilityQosPolicyKind kind;
+		Duration_t max_blocking_time;
+	};
+
+The equivalent representation in XML is as follows:
+
+.. code-block:: xml
+
+	<reliability>
+      <kind></kind>
+      <max_blocking_time>
+          <sec></sec>
+          <nanosec></nanosec>
+      </max_blocking_time>
+	</reliability>
+
+Sequences
+---------
+
+In general, the sequences contained in the QoS policies are described with the following XML format:
+
+.. code-block:: xml
+
+	<a_sequence_member_name>
+      <element>...</element>
+      <element>...</element>
+      ...
+	</a_sequence_member_name>
+
+Each element of the sequence is enclosed in an <element> tag, as shown in the following example:
+
+.. code-block:: xml
+
+	<property>
+      <value>
+          <element>
+              <name>my name</name>
+              <value>my value</value>
+          </element>
+          <element>
+              <name>my name2</name>
+              <value>my value2</value>
+          </element>
+      </value>
+	</property>
+
+
+Enumerations
+------------
+
+Enumeration values are represented using their IDL string representation. For example:
+
+.. code-block:: xml
+
+    <history>
+        <kind>KEEP_ALL_HISTORY_QOS</kind>
+    </history>
+
+
+Time values (Durations)
+-----------------------
+
+Following values can be used for fields that require seconds or nanoseconds:
+
+- DURATION_INFINITE_SEC
+- DURATION_INFINITE_NSEC
+
+The following example shows the use of time values:
+
+.. code-block:: xml
+
+    <deadline>
+        <period>
+            <sec>DURATION_INFINITE_SEC</sec>
+            <nanosec>DURATION_INFINITE_NSEC</nanosec>
+        </period>
+    </deadline>
+
+QoS Profiles
+------------
+
+A QoS profile groups a set of related QoS, usually one per entity. For example:
+
+.. code-block:: xml
+
+    <qos_profile name="StrictReliableCommunicationProfile">
+        <datawriter_qos>
+            <history>
+                <kind>KEEP_ALL_HISTORY_QOS</kind>
+            </history>
+            <reliability>
+                <kind>RELIABLE_RELIABILITY_QOS</kind>
+            </reliability>
+        </datawriter_qos>
+        <datareader_qos>
+            <history>
+                <kind>KEEP_ALL_HISTORY_QOS</kind>
+            </history>
+            <reliability>
+                <kind>RELIABLE_RELIABILITY_QOS</kind>
+            </reliability>
+        </datareader_qos>
+    </qos_profile>
+
+XML Example
+-----------
+
+Consider the following XML file that describes two QoS profiles:
+
+- FooQosProfile
+    - DataWriterQos - Transient
```

Comment:
I would pick some QoS other than durability because we don't yet support transient/persistent (although transient-local would be fine). It doesn't matter much which it is, as long as everything is fully supported.